rollup.config.js

```diff
@@ -27,9 +27,7 @@ const config = {
     external(),
     postcss({
       extract: true,
-      modules: {
-        globalModulePaths: ["src/components/Select/styles.css"],
```

Comment:
wondering why we had this here, as `src/components/Select/styles.css` file didn't exist before this commit 🤔

Reply:
It didn't exist before this commit, but it did exist _before this PR_. 
The `globalModulePaths` config was introduced by: https://github.com/decred/pi-ui/pull/227

Reply:
oh, i see, makes sense.

it was imported then as plain .css as it included pure css classes prefixed with `customSelect` which isn't
the case anymore, glad it evolved to a css module. 👍🏽